database/migrations/create_bearer_table.php.stub

```diff
@@ -11,6 +11,7 @@ return new class extends Migration
         Schema::create('bearer_tokens', function (Blueprint $table) {
             $table->id();
             $table->string('token')->unique();
+            $table->text('description')->nullable();
```

Comment:
This should probably be a varchar rather than text.

Reply:
Yeah, that also works great. But this introduces a maximum of 255 characters I think. Is it alright to include that in the readme?

Reply:
@ryangjchandler kindly take a look at the update to the PR.